app/models/tutorial.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Tutorial < ApplicationRecord
+  validates_presence_of :title, :description, :thumbnail
+
```

Comment:
These validations prevent tutorial creation if new form is filled out incorrectly.